Sources/ApiVideoPlayer/View/ApiVideoPlayer.swift

```diff
@@ -0,0 +1,39 @@
+#if !os(macOS)
+import SwiftUI
+
+@available(iOS 14, macOS 11.0, *)
+public struct ApiVideoPlayer: UIViewControllerRepresentable {
+
+  private let videoId: String
+  private let videoType: VideoType
+  private let viewPlayer: SwiftUIPlayerViewController
+
+  public init(videoId: String, videoType: VideoType, events: PlayerEvents? = nil) {
+    self.videoId = videoId
+    self.videoType = videoType
+    self.viewPlayer = SwiftUIPlayerViewController(videoId: videoId, videoType: videoType, events: events)
+  }
+
+  public func makeUIViewController(context _: Context) -> SwiftUIPlayerViewController {
+    return self.viewPlayer
+  }
+
+  public func updateUIViewController(_: SwiftUIPlayerViewController, context _: Context) {}
+
+  public func play() {
+    self.viewPlayer.play()
+  }
+
+  public func pause() {
+    self.viewPlayer.pause()
+  }
```

Comment:
Is there a reason why there is only `play` and `pause` here? What about `seek`,...